### PR TITLE
PLAT-34: Set the default value of the Login url Global Property as th…

### PR DESCRIPTION
**Pull request by mozzy11**
Ticket : https://issues.openmrs.org/browse/PLAT-34

### CODE REVIEW

File: api/src/main/java/org/openmrs/util/OpenmrsConstants.java [hunk 1]
```diff
@@ -1307,6 +1308,13 @@ public static String getOpenmrsProperty(String property) {
 	/** Value for the long person name format */
 	public static final String PERSON_NAME_FORMAT_LONG = "long";
 	
+	public static final String PLATFORM_UI_LOGIN_URL = "owa/openmrs-owa-platformui-0.1.0/index.html#/login";
+	
+	public void setDefaultLoginUrl_GP() {
+	Context.getAdministrationService().getGlobalPropertyObject(GP_LOGIN_URL).setPropertyValue(PLATFORM_UI_LOGIN_URL);
+	}
+
```
Comment (Ruhanga) [inline api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: Thank you @mozzy11 for the work. Did you test this?

Comment (mozzy11) [reply api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: @Ruhanga  , you mean testing it on my local installtion?? yes i did

Comment (Ruhanga) [reply api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: Can you also attach screen shots reflecting your changes as advised here https://wiki.openmrs.org/display/docs/Pull+Request+Tips ?

Comment (mozzy11) [reply api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: hi @Ruhanga , i made a small update in the PR , thanks . i have attached the image showing the default Global property value changed to point to the platform-ui Login url

Comment (mozzy11) [reply api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: updated commit is here  https://github.com/openmrs/openmrs-core/pull/2841/commits/5907c1b9806582a74dc88957ab6d95ce3ae11b95

Comment (Ruhanga) [reply api/src/main/java/org/openmrs/util/OpenmrsConstants.java]: @mozzy11 , what happens when a User logs in with the set set Login url? How does the platform UI login page look like? What happens after a user logs in?

**Comment by mozzy11** [comment]
@Ruhanga , the platform-ui is a web app that is just installed in the platform to provide a UI on the platform without the Needing the Legacy Hi. Unless the user has the web app installed in their platform , that URL will point to no page at all. But any way the ticket required I change the default Login-url global property to the platform _ui_login url , which I have done , because the initial default value has been pointing to the usual Login Url of the Legacy UI

**Comment by Ruhanga** [comment]
Do your changes make any effect? If so, where are the visual effects of this in action given a platform instance created with use-case you have mentioned?

**Comment by mozzy11** [comment]
> Do your changes make any effect? If so, where are the visual effects of this in action given a platform instance created with use-case you have mentioned?

@Ruhanga , this does not necesarilly make any visal changes , it was a matter of setting a default value of the Login_url Global property to have a default value of the platorm-ui Login URL.  the issue is icase any one wants to use that global property , it will have a default value of the platform-ui login URl.

**Comment by mozzy11** [comment]
When you take a keen look at the image i attached on the ticket , youll discover that the default value of the Login_url Global property is now the platform-ui Login url as required by the ticket

**Comment by Ruhanga** [comment]
Does this mean the default platform url being used leads to a dead end or links to a page that doesn't exist or can't be diplayed?

**Comment by mozzy11** [comment]
@Ruhanga , noo , this is not a url being used. Actually as for now , in all the platform code , they refer to the login url as ` request.getContextPath() + "/login.htm"`    , so its not that the value i have set is used any where as for now .  the Login_url is actually a new Global property that was created not long ago . so wat i set was just its default value, just in case any implementation needs to use the platform-ui Login url , they can acces it from that global property , but not that its currently being used in code

**Comment by mozzy11** [comment]
in other words , this is just a global property value , since ,in the time to come , we plan to completely use react web apps instead of the legacy ui , that global property would probably be very useful

**Comment by mozzy11** [comment]
i updated the PR , so that the platform-ui login url is

owa/openmrs-owa-platformui-0.1.0/index.html#/login.htm
not

 owa/openmrs-owa-platformui-0.1.0/index.html#/login
i had forgorten to append the .htm at the end

**Comment by mozzy11** [comment]
@Ruhanga  , i attached the image of the owa-paltform-ui Login page on the ticket. you can take a look at it.

**Pull request closed without merging**